src/WCSimVisManager.cc

```diff
@@ -141,7 +141,8 @@ void WCSimVisManager::RegisterGraphicsSystems () {
     G4cout <<
       "\nYou have successfully chosen to use the following graphics systems."
 	 << G4endl;
-    PrintAvailableGraphicsSystems ();
+    //PrintAvailableGraphicsSystems (); //use this version for Geant4.10.1
```

Comment:
Split to separate PR (using precompiler and Geant version environment variable if possible)

Reply:
resolved in #10